Mathlib/CategoryTheory/SmallObject/Iteration/Basic.lean

```diff
@@ -48,7 +48,7 @@ file `SmallObject.TransfiniteIteration`.
 
 The map `Φ.toSucc X : X ⟶ Φ.succ X` does not have to be natural
 (and it is not in certain applications). Then, two isomorphic
-objects `X` and `Y` may have non isomorphic successors. This is
+objects `X` and `Y` may have nonisomorphic successors. This is
```

Comment:
I've never seen this un-hyphenised. Same as above.

Reply:
It seems non-isomorphic might be more common, so I might be a bit less militant on this one. There is this one, though: https://mathworld.wolfram.com/Nonisomorphic.html

Also the general considerations listed in the comment below.

Reply:
Then I'd say, let's go with the more common spelling.

Reply:
Done.